### DISCUSSION TITLE
How to transfer data to another computer?

### DISCUSSION THREAD

**Discussion by noroi110** [MalloyDelacroix/DownloaderForReddit#339] [Q&A]
I've downloaded quite a lot from my previous laptop, would have loved to transfer the database to my new computer so I can avoid all the duplicate posts. Thanks!

**Comment by MalloyDelacroix** [comment]
The database file is located in the application data folder.  On Windows its at: "C:/Users/<user>/AppData/Roaming/SomeGuySoftware/DownloaderForReddit".  You can open this directory from the app by going to File > Open Data Directory.  

The database file is called dfr.db.  Just copy this file to the new location.  It should all be self contained.  If you want to keep some of your other application settings not related to data stored in the database, you can also move the config.toml file.  This file controls all of the appearance and performance settings for the app.

**Reply by noroi110** [reply]
Thank you! It appeared in my new device now.
